packages/broker/src/plugins/operator/MaintainTopologyHelper.ts

```diff
@@ -6,7 +6,7 @@ import { EventEmitter } from "eventemitter3"
 import { FetchResponse, Logger, TheGraphClient } from "@streamr/utils"
 
 /**
- * Events emitted by {@link OperatorClient}.
+ * Events emitted by {@link MaintainTopologyHelper}.
  */
 export interface OperatorClientEvents {
```

Comment:
For consistency I would name:
```suggestion
export interface MaintainTopologyHelperEvents {
```